docs/customizing.md

```diff
@@ -245,3 +245,271 @@ Example `pulp_install_plugins` (with bogus version values):
+  vars:
+    pulp_default_admin_password: << YOUR PASSWORD FOR THE PULP APPLICATION HERE >>
+    pulp_api_bind: example-pulp-api-server:24817
+    pulp_settings:
+      secret_key: << YOUR SECRET HERE >>
+      content_origin: "http://exanple-webserver.fqdn"
+      redis_host: example-redis-server
+      redis_port: 6379
+      databases:
+        default:
+          HOST: example-postgres-server
+          ENGINE: django.db.backends.postgresql_psycopg2
+          NAME: pulp
+          USER: pulp
+          PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+    pulp_install_plugins:
+      pulp-container: {}
+      pulp-rpm: {}
+  roles:
+    - pulp_api
+    - pulp_database_config
+  environment:
+    DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+
+- hosts: example-pulp-content-server
+  vars:
+    pulp_content_bind: example-pulp-content-server:24816
+    pulp_settings:
+      secret_key: << YOUR SECRET HERE >>
+      content_origin: "http://exanple-webserver.fqdn"
+      redis_host: example-redis-server
+      redis_port: 6379
+      databases:
+        default:
+          HOST: example-postgres-server
+          ENGINE: django.db.backends.postgresql_psycopg2
+          NAME: pulp
+          USER: pulp
+          PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+    pulp_install_plugins:
+      pulp-container: {}
+      pulp-rpm: {}
+ roles:
+   - pulp-content
+ environment:
+   DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+
+- hosts:
+   - example-pulp-worker-server1
+   - example-pulp-worker-server2
+ vars:
+   pulp_settings:
+     secret_key: << YOUR SECRET HERE >>
+     content_origin: "http://exanple-webserver.fqdn"
+     redis_host: example-redis-server
+     redis_port: 6379
+     databases:
+       default:
+         HOST: example-postgres-server
+         ENGINE: django.db.backends.postgresql_psycopg2
+         NAME: pulp
+         USER: pulp
+         PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+   pulp_install_plugins:
+     pulp-container: {}
+     pulp-rpm: {}
+ roles:
+   - pulp_workers
+ environment:
+   DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+
+- hosts: example-resource-manager-server
+ vars:
+   pulp_settings:
+     secret_key: << YOUR SECRET HERE >>
+     content_origin: "http://exanple-webserver.fqdn"
+     redis_host: example-redis-server
+     redis_port: 6379
+     databases:
+       default:
+         HOST: example-postgres-server
+         ENGINE: django.db.backends.postgresql_psycopg2
+         NAME: pulp
+         USER: pulp
+         PASSWORD: << YOUR DATABASE PASSWORD HERE >>
+   pulp_install_plugins:
+     pulp-container: {}
+     pulp-rpm: {}
+ roles:
+   - pulp_resource_manager
+ environment:
+   DJANGO_SETTINGS_MODULE: pulpcore.app.settings
+
+- hosts: exanple-webserver
+ vars:
+   pulp_content_bind: example-pulp-server:24816
+   pulp_api_bind: example-pulp-server:24817
+ roles:
+   - pulp_webserver
+
+```
+
+Note that the `example-pulp-api-server` is also used to to run `pulp_database_config`.
```

Comment:
Which will do what?
Run migrations?

Reply:
I paraphrased from this: 

"pulp-api1: The pulp-api server. We also chose this to run pulp_database_config."

Reply:
And i believe it will create and migrate the database for pulp.
It should probably be limited to a single node (in case you want multiple api-workers).

@mikedep333 Do you think `run_once: true` can be used?

Reply:
I will wait for @mikedep333  on this one because I really don't understand what is happening here.

Reply:
@mdellweg Yeah, that might be a good approach to do instead.

Let's discuss this as a separate change.

Reply:
The "pulp-api1" name is the ansible standard. Because there can be more later on.

If you're going for clarity, this would be better though.

Reply:
I changed back to pulp-api1 and added a note that this was an ansible standard, that one implies the possibility of two etc. 
I added a *bit* of what @mdellweg  said but don't really know how to get it right and would be happy if either someone explicitly told me what to say or it was handled in a separate PR.

Reply:
Very sorry, I decided we should change it back. To be consistent with the rest of the document. I didn't realize this until just now. (I did a partial review only previously when I replied to mdellweg's comment.)